frame/system/src/lib.rs

```diff
@@ -286,6 +274,11 @@ pub trait Trait: 'static + Eq + Clone {
 	type OnKilledAccount: OnKilledAccount<Self::AccountId>;
 
 	type SystemWeightInfo: WeightInfo;
+
+	/// Return block weights object.
+	fn block_weights() -> limits::BlockWeights {
```

Comment:
nothings against it but wondering if it is just a helper or is it solving any problems?

Reply:
All good as it is.